addons/popochiu/engine/interfaces/i_audio.gd

```diff
@@ -22,6 +22,13 @@ extends Node
 var twelfth_root_of_two := pow(2, (1.0 / 12))
 
 
+#region Godot ######################################################################################
+func _init() -> void:
+	Engine.register_singleton(&"A", self)
```

Comment:
Interesting! Mind explaining why are you doing it here?

Reply:
Because this is the function called by Godot when an object is initialized. That way we assure the first thing the object does is to register itself as a Singleton. To be honest, a saw this pattern in another plugin and I liked it. Before this PR we were not registering our Global classes as singletons.